docs/setup-cluster/on-prem/options/linux-packages.rst

```diff
@@ -137,12 +137,18 @@ Install the Determined Master and Agent
 
    **Red Hat Distributions**
 
-   On Red Hat distributions, use the following command:
+   On Red Hat distributions, use the following command during the initial installation:
 
       .. code::
 
          sudo rpm -i <path to downloaded package>
 
+   When upgrading, use the following command:
```

Comment:
This is fine, but the issue with trying to integrate it here is that they might miss the "Upgrade the Cluster" step way down on this page and it will leave the cluster in a non-functional state.
It would seem clearer to leave this section as initial install, and provide the rpm/apt-install commands in the Upgrade section below instead so that the steps to restart the cluster are part of the process and not tacked on and easily missed.

Reply:
agree

Reply:
resolved